### PR TITLE
Add row type (delete/insert/update) to fallback error message

### PR DESCRIPTION
**Pull request by ThuTrinh**
@zendesk/goanna 
Adding row type change to the fallback message.

JIRA: https://zendesk.atlassian.net/browse/MELINF-982

Screen Shot 2019-07-30 at 3 18 56 pm

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/row/RowIdentity.java [hunk 1]
```diff
@@ -10,12 +10,14 @@
 public class RowIdentity implements Serializable {
 	private String database;
 	private String table;
+	private String rowType;
```
Comment (lukestephenson-zendesk) [inline src/main/java/com/zendesk/maxwell/row/RowIdentity.java]: can this be tagged `final`?

Comment (ThuTrinh) [reply src/main/java/com/zendesk/maxwell/row/RowIdentity.java]: just checked usages, all three can :)

**Comment by lukestephenson-zendesk** [comment]
This looks OK, but the tests are failing.  Not sure if it's related to this change.

**Comment by ThuTrinh** [comment]
Looks like the test failures are due to a separate issue related to bootstrapping. I can see similar tests failing in other PRs.

**Pull request merged**